pkg/controller/wavefrontupgradeutil/upgrade.go

```diff
@@ -0,0 +1,74 @@
+package wavefrontupgradeutil
+
+import (
+	"net/http"
+	"github.com/tidwall/gjson"
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"github.com/prometheus/common/log"
+	"github.com/Masterminds/semver"
+)
+
+const (
+	ProxyImageName = "proxy"
+
+	CollectorImageName = "wavefront-kubernetes-collector"
+)
+
+func GetLatestVersion(crImageName string, currentVersion string, enableAutoUpgrade bool) (string, error) {
```

Comment:
Can you add some unit tests around this? And also add a `.travis.yml` file for this repo that runs the unit tests.